apps/condo/domains/billing/schema/BillingAccount.test.js

```diff
@@ -2,404 +2,467 @@
-const { makeClientWithIntegrationAccess } = require('@condo/domains/billing/utils/testSchema')
-const { OrganizationEmployee, updateTestOrganizationEmployee } = require('@condo/domains/organization/utils/testSchema')
-const { makeClientWithNewRegisteredAndLoggedInUser } = require('@condo/domains/user/utils/testSchema')
-const { makeLoggedInAdminClient, makeClient } = require('@core/keystone/test.utils')
-const { BillingAccount, createTestBillingAccount,
-    updateTestBillingAccount, createTestBillingProperty,
-    makeContextWithOrganizationAndIntegrationAsAdmin, createTestBillingIntegrationOrganizationContext,
-    makeOrganizationIntegrationManager, createReceiptsReader } = require('@condo/domains/billing/utils/testSchema')
-const { expectToThrowAuthenticationErrorToObjects, expectToThrowAccessDeniedErrorToObj, expectToThrowAuthenticationErrorToObj } = require('@condo/domains/common/utils/testSchema')
-const { catchErrorFrom } = require('@condo/domains/common/utils/testSchema')
+const dayjs = require('dayjs')
+const {
+    makeServiceUserForIntegration,
+    makeOrganizationIntegrationManager,
+    makeContextWithOrganizationAndIntegrationAsAdmin,
+    createReceiptsReader,
+    createTestBillingProperty,
+    BillingAccount,
+    createTestBillingAccount,
+    createTestBillingAccounts,
+    updateTestBillingAccount,
+    updateTestBillingAccounts,
+} = require('@condo/domains/billing/utils/testSchema')
+const {
+    makeClientWithNewRegisteredAndLoggedInUser,
+    makeClientWithSupportUser,
+} = require('@condo/domains/user/utils/testSchema')
+const {
+    expectToThrowAuthenticationErrorToObj,
+    expectToThrowAuthenticationErrorToObjects,
+    expectToThrowAccessDeniedErrorToObj,
+    expectToThrowAccessDeniedErrorToObjects,
+    expectToThrowInternalError,
+} = require('@condo/domains/common/utils/testSchema')
+const { makeClient } = require('@core/keystone/test.utils')
 
 describe('BillingAccount', () => {
-    describe('Constraints', () => {
-        test('can create two BillingAccount with same globalId in different contexts', async () => {
-            const admin = await makeLoggedInAdminClient()
-            const { context } = await makeContextWithOrganizationAndIntegrationAsAdmin()
-            const [property] = await createTestBillingProperty(admin, context)
-            const [billingAccount] = await createTestBillingAccount(admin, context, property, {
-                globalId: 'cat',
+    let admin
+    let support
+    let anonymous
+    let user
+    let context
+    let property
+    let integrationUser
+    let integrationManager
+    let anotherContext
+    let anotherProperty
+    beforeAll(async () => {
+        const { admin: adminClient, context: billingContext, integration } = await makeContextWithOrganizationAndIntegrationAsAdmin()
+        admin = adminClient
+        context = billingContext
+        const [firstProperty] = await createTestBillingProperty(admin, context)
```

Comment:
I'm just recreating common state before modifying accounts. It's not a big deal here, since we only need to form correct relations. But I can change it to integration manager if you insist